website/blog/2023-02-24-Deploy-databend-on-minio.md

```diff
@@ -0,0 +1,211 @@
+| - databend-version-nightly-aarch64-unknown-linux-musl.tar.gz | Linux of Arm  |
+| - databend-version-nightly-x86_64-apple-darwin.tar.gz        | MacOS of X64  |
+| - source code                                                | src           |
+
+We can download the musl package for Linux from the Databend open-source project repo a: https://github.com/datafuselabs/databend/tags
+
+
+Generally, we can download the musl package for Linux. Note the difference between arm and x86_64 platforms.
+
+Download: https://github.com/datafuselabs/databend/tags
+
+For version selection, it is recommended to download the latest version every day. Databend is fast to develop and many new features are merged quickly.For example, if you're running Linux on an x86_64 machine, you can download version v0.9.49 like this:
+
+```Bash
+cd /data
+mkdir databend
+export ver=v0.9.49
+wget https://repo.databend.rs/databend/$ver-nightly/databend-$ver-nightly-x86_64-unknown-linux-musl.tar.gz
+cd databend
+tar zxvf ../databend-$ver-nightly-x86_64-unknown-linux-musl.tar.gz
+```
+
+![ ](../static/img/blog/databend-minio-beginner-01-3.png)
+
+Databend installation files extracted a directory named "databend". 
+
+### Configure Databend
+
+Once you have installed Databend, you'll need to configure it. The default configuration file for databend-query is included with the download. You can modify this file as follows:
+
+```Bash
+vim configs/databend-query.toml
+```
+
+change:
+
+```Bash
+# Storage config.[storage]
+# fs | s3 | azblob | obs
+type = "s3"
+# Set a local folder to store your data.
+# Comment out this block if you're NOT using local file system as storage.
+[storage.fs]data_path = "./.databend/stateless_test_data"
+# To use S3-compatible object storage, uncomment this block and set your values.
+[storage.s3]
+bucket = "databend"
+endpoint_url = "https://127.0.0.1:9900"
+access_key_id = "minioadmin"
+secret_access_key = "minioadmin"
+```
+
+### Start and Stop Databend 
+
+To start Databend, run the following command:
+
+```Bash
+./script/start.sh
+```
+
+To check that Databend is running, you can use the following command:
+```
+ps axu |grep databend
+```
+
+![](../static/img/blog/databend-minio-beginner-01-4.png)
+
+To stop Databend, run the following command:
+
+```Bash
+./script/stop.sh 
+```
+
+#### Connect to  Databend
+
+Databend has three external service ports by default: 
+ 
+- MySQL: 3307 supports MySQL cli and application connection. 
+- Clickhouse: 8124 Clickhouse http handler protocol 
+- [http prot: 8000](https://databend.rs/doc/integrations/api/rest) The Databend HTTP handler is a REST API
+ 
+To connect to Databend using the MySQL client, run the following command:
+
+```Bash
+mysql -h 127.0.0.1 -P3307 -uroot
+```
+
+Note that root can login without a password using localhost. Databend permission management refers to the design of MySQL 8.0 and allows you to manage Databend users in the same way as MySQL 8.0 user management. 
```

Comment:
Note that root can login without a password using localhost. -> Note that the root user can log in without a password from localhost.